web/templates/cluster.html.tmpl

```diff
@@ -7,13 +7,36 @@
       <dd class="inline">{{ .Cluster.Crmmon.Summary.Nodes.Number }}</dd>
       <dt class="inline">Resource number</dt>
       <dd class="inline">{{ .Cluster.Crmmon.Summary.Resources.Number }}</dd>
+      <dt class="inline">Fencing type</dt>
+      <dd class="inline">{{ .Cluster.FencingType }}</a></dd>
       {{- range .Cluster.Cib.Configuration.CrmConfig.ClusterProperties }}
       <dt class="inline">{{ .Id }}</dt>
       <dd class="inline">{{ .Value }}</dd>
       {{- end }}
     </dl>
+    {{- if not .Cluster.IsFencingEnabled }}
+    <div class="alert alert-section alert-danger">
+      <i class="eos-icons eos-18 alert-icon">danger</i>
+      <div class="alert-body">
+        <div class="alert-title">Fencing disabled</div>
+        Fencing is not enabled in the cluster. Fencing is mandatory for a correct functioning of the cluster.</br>Find more information <a href="https://documentation.suse.com/sle-ha/15-SP1/single-html/SLE-HA-guide/#sec-ha-requirements-hw" target="_blank">here</a>.
+      </div>
+    </div>
+    {{- end }}
+    {{- if not .Cluster.FencingResourceExists }}
+    <div class="alert alert-section alert-danger">
+      <i class="eos-icons eos-18 alert-icon">danger</i>
+      <div class="alert-body">
+        <div class="alert-title">Fencing resource not found</div>
+        There is not any fencing resource configured. A fencing resource is mandatory for a correct functioning of the cluster.</br>Find more information <a href="https://documentation.suse.com/sle-ha/15-SP1/single-html/SLE-HA-guide/#sec-ha-requirements-hw" target="_blank">here</a>.
+      </div>
+    </div>
+    {{- end }}
+    {{- if .Cluster.IsFencingSBD }}
+    {{ template "sbd" .Cluster.SBD }}
     <hr/>
     <p class='clearfix'/>
+    {{- end }}
```

Comment:
Just a suggestion: I would put the entire SBD section after the Nodes one, which I think is the most important one.

Reply:
Done: https://github.com/trento-project/trento/pull/72/commits/adbc734cbac7bf9168c7e68e0904f8a6f21eb731